code/common/pubsub.q

```diff
@@ -174,9 +174,15 @@ init:{[t]
 
 // Initialise subscription request on startup
 .ds.subreq:(`u#`$())!`int$();
+
 // Striping function which stores the mappings for any symbols that it has already computed and 
 // for subsequent requests for that symbol, it looks them up
-.ds.stripe:{[input;skey]
+.ds.stripe:{[input]
+  // Define number of segments and match each segid to a section
+  .ds.numseg:`int$count .ds.segmentlist;
+  .ds.segsect:.ds.segmentlist!til count .ds.segmentlist;
+  // Define segment based on sym split
+  skey:first .ds.segsect `$.proc.params[`segid];
```

Comment:
I could be mistaken, but I thought we got rid of the `-segid` command line param? Are we re-introducing it? Or has it always been there still?

Reply:
No, I believe we still use the segid, it's the .ds.datastripe variable that we got rid of in the command line. We may be getting rid of it in future if we sort out the extras to the json and such like but it's still in for now.